OutOfSchool/OutOfSchool.WebApi/Controllers/V1/FeatureFlagsController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.FeatureManagement;
+using Microsoft.FeatureManagement.Mvc;
+using OutOfSchool.WebApi.Models;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Linq;
+using OutOfSchool.WebApi.Enums;
+
+namespace OutOfSchool.WebApi.Controllers.V1
+{
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Route("api/v{version:apiVersion}/[controller]/[action]")]
+    public class FeatureFlagsController : ControllerBase
+    {
+        private readonly IConfiguration _config;
+
+        public FeatureFlagsController(IConfiguration configuration, ILogger<FeatureFlagsController> logger)
+        {
+            _config = configuration;
+        }
+
+        /// <summary>
+        /// Get enables feature flags depending on releases.
+        /// </summary>
+        /// <returns>List of releases.</returns>
+        /// <response code="200">All entities were found.</response>
+        /// <response code="204">No entity was found.</response>
+        /// <response code="500">If any server error occures.</response>
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(IEnumerable<Feature>))]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        [HttpGet]
+        public IActionResult Get()
+        {
+            var releases = _config.GetSection("FeatureManagement").GetChildren().Where(x => x.Value.Equals("True")).ToList();
```

Comment:
Please create a dedicated class `FeatureManagementConfig` and move all related things there.
Also add a field:
 `public const string SectionName = "FeatureManagement"`
to that class and use that field instead of literal here an in all places below.